user-interface/3001-VOTE-vote.md

```diff
@@ -1 +1,157 @@
+
+Each proposal type will have a majority required (vote weight in favour) and a participation required (the turn out for the vote needs to be sufficient to be valid).
+
+A proposal has a life cycle and various statuses to track it's progress. proposals that accept votes can have a few different statuses. A short hand is used in these ACs:
+
+- Open = Accepting votes (includes `waitingForNodeVote`)
+- To enact = passed but not yet enacted (`pending)
+- Closed = was accepting votes but deadline has passed (e.g. `passed`, `rejected` etc)
+- `Failed` = did not get to the point of accepting votes.
+
+## list of proposals
+
+When looking for a particular proposal or wanting to see what proposals are open, soon to enact or closed, I...
+
+- **must** see link to details on how governance works in docs (<a name="3001-VOTE-001" href="#3001-VOTE-001">3001-VOTE-001</a>)
+- **must** see link(s) to make proposals (<a name="3001-VOTE-002" href="#3001-VOTE-002">3001-VOTE-002</a>)
+- **must** if there are no proposals, see that there have been no proposals since the last chain checkpoint restore (<a name="3001-VOTE-003" href="#3001-VOTE-003">3001-VOTE-003</a>)
+- **must** see open proposals (and ones due for enactment) distinct from others (e.g grouped by "open", "to enact" "closed") (note: freeform proposals do not enact so should be shown as "closed" when "passed") (<a name="3001-VOTE-004" href="#3001-VOTE-004">3001-VOTE-004</a>)
+- **should** see proposals sorted with the ones closest to enactment first (within each group) 
+- **must** see a history of all "closed" proposals (<a name="3001-VOTE-006" href="#3001-VOTE-006">3001-VOTE-006</a>)
+- can search for a proposal. by:
+  - **should** be able to search by proposal ID
+  - **should** be able to search by public key of the proposer
+  - **should** be abel to search by market ID/name/code (ID may be the same as proposal ID)
+  - **should** be able to search by asset name/symbol
+  - **should** be able to search by network parameter
+  - **should** be able to search by content of proposal description
+
+for each proposal:
+
+- **must** see the type of proposal (<a name="3001-VOTE-007" href="#3001-VOTE-007">3001-VOTE-007</a>)
+- **must** see the proposal title (<a name="3001-VOTE-097" href="#3001-VOTE-097">3001-VOTE-097</a>)
+- **should** see a summary of what the type of proposed change is, without looking at details (network, new market etc)
+  - for network parameters: **should** see what parameter is being changed and new value
+  - for network parameters: **could** see what the current values are for that parameter
+  - for network parameters: **could** see if there are other open proposals for the same parameter
+  - for new markets: **should** see the type of market (e.g. Future)
+  - for new markets: **could** see the type trading mode of the market (e.g. auction, continuous)
```

Comment:
nitpick, can we move this down so ACs are in descending order of importance

Reply:
I'd rather not. rationale: to have a single order we list proposal types and hence make them more findable